src/forms/NewJobForm/index.jsx

```diff
@@ -93,6 +106,31 @@ export const NewJobForm = ({ token, coinSymbol }) => {
     !isEmptyString(description) &&
     !isEmptyString(budget)
 
+  const renderWriteJob = () => {
+    return (
+      <Form.Input
+        control={TextArea}
+        id="description"
+        label="Description"
+        placeholder=""
+        rows={12}
+        value={description}
+        onChange={handleDescriptionChange}
+        required
+      />
+    )
+  }
+
+  const renderPreviewJob = () => {
+    return (
+      <ReactMarkdown>
```

Comment:
I think it would be good to use existing `<FormattedDescription />` component from `./src/components` directory (and inside `EditJobForm`). Can you please make these changes?

JFYI: I disabled some tags like Headings and blockquotes from `ReactMarkdown` to skip it on rendering.

Reply:
Sure, makes sense! If you have more suggestions, feel free to add here in comments!

Reply:
Everything else is great! 👍 

I'll refactor `EditJobForm` later (regarding existing changes in `NewJobForm` about `localStorage`)

Reply:
What do you mean by "these tags break markup"? I think it worked well when I tried doing it.
For example, when you put an header, you must put two breaklines, in order for the text to appear on the next line. If only you could explain me better, because I think users can misunderstand why headers don't work, unless you mention it somewhere.

Reply:
## Point 1

> "These tags break markup"

I am not sure about allowing customers to use headings in these forms because they will use it in incorrect way 100%. Of course I think about SEO optimizations and too many H1-H2 headers will work against us.

If you try to use job creation form and put something like this:

```markdown
# Header 1
# Header 1 Again
# Another Header 1
```

It will affect SEO for this specific job page.

## Point 2

> because I think users can misunderstand why headers don't work, unless you mention it somewhere.

We mentioned it on the same form which you've changed today :-D

> You can use [Markdown ](https://commonmark.org/help/)formatting elements in a description.
> Headings (h1-h6) and blockquote are not supported.

Reply:
@sergiimyk, hi!

Can you please fix linter errors and push your changes?
I would like to deploy this feature to production today :-)

Reply:
Hi, @gruz0 

Sure thing!